packages/styles/src/modal/modal.scss

```diff
@@ -1,19 +1,23 @@
 @import "../core/_dev";
 
 .as-modal {
+  --as-modal-background: rgba(44, 44, 44, 0.8);
+
   display: flex;
   position: absolute;
+  z-index: 2;
```

Comment:
Is `2` enough? should we use a higher value?

Reply:
which value is ok?

Reply:
I have no idea, but we're going to have the same problem with dropdowns, so we should define a list of z-index(es)